docs/src/documentation/03-components/02-structure/tabs/03-accessibility.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/tabs/accessibility/
+---
+
+# Accessibility
+
+## Tabs
+
+The Tabs component has been designed with accessibility in mind.
+
+### Accessibility props
+
+The following props are available to improve the accessibility of your Tabs component:
+
+#### TabList
+
+| Prop           | Type     | Description                                                    |
+| :------------- | :------- | :------------------------------------------------------------- |
+| ariaLabel      | `string` | Allows you to provide an accessible label for the set of tabs. |
+| ariaLabelledby | `string` | References the ID of an element that labels the set of tabs.   |
+
+The `ariaLabel` prop should ideally be translated and descriptive of the purpose of the tab set. It should clearly communicate the type of content users will find within the tabs.
+
+The `ariaLabelledby` prop is used when you already have a visible heading that describes the tab set, allowing you to reuse that text for screen readers.
+
+#### Tab
+
+| Prop     | Type      | Description                                                        |
+| :------- | :-------- | :----------------------------------------------------------------- |
+| disabled | `boolean` | Indicates that the tab is not interactive and cannot be activated. |
+
+### Accessibility implementation
+
+The Tabs component automatically implements proper accessibility features including:
+
+- Correct ARIA roles: `tablist` for the TabList, `tab` for individual tabs, and `tabpanel` for tab panels
+- Appropriate ARIA states: `aria-selected`, `aria-disabled`, `aria-controls`, and `aria-labelledby`
```

Comment:
Imo we should clarify this `aria-labelledby`. We should say it refers to the Panel knowing the Tab that labels it, or something like that. So that it is not confused with the `ariaLabelledby` prop